### PR TITLE
Update 3.1.1

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge/staged-recipes  ready for review

**Review by darynwhite — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +45,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - cshaley
```
Comment (ocefpaf) [inline recipe/meta.yaml:45]: You should not remove maintainers, even inactive ones, without their consent.

Comment (darynwhite) [reply recipe/meta.yaml:45]: #19 should fix this; thanks for reviewing our edits!

**Pull request merged**